smartsim/_core/generation/generator.py

```diff
@@ -154,89 +177,82 @@ def _gen_exp_dir(self) -> None:
             dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
             log_file.write(f"Generation start date and time: {dt_string}\n")
 
-    def _gen_feature_store_dir(self, feature_store_list: t.List[FeatureStore]) -> None:
-        """Create the directory that will hold the error, output and
-           configuration files for the feature store.
+        # Prevent access to type FeatureStore entities
```

Comment:
This point is probably moot as we are already going to _severely_ overhaul or outright remove the `FeatureStore` as it is today, so don't feel the need to act on it, but I am curious: Why wouldn't we want to allow for generation of the `FeatureStore`? Doesn't that mean that its output and error files will end up in the cwd??

Reply:
I had this blocker here since FeatureStore does not have a files attr!